packages/experimental/Link/src/useLink.ts

```diff
@@ -44,7 +44,7 @@ export const useLink = (props: LinkProps): LinkInfo => {
   // GH #1336: Set focusRef to null if link is disabled to prevent getting keyboard focus.
   const focusRef = isDisabled && !focusable ? null : componentRef;
   const onPressWithFocus = useOnPressWithFocus(focusRef, linkOnPress);
-  const pressable = useAsPressable({ ...rest, disabled: isDisabled, onPress: onPressWithFocus });
+  const pressable = usePressableState({ ...rest, disabled: isDisabled, onPress: onPressWithFocus });
```

Comment:
I saw elsewhere in Text.tsx that you added a call to `useAsPressable`.I don't think the two should be mixed. Is the intention of this PR to use `useAsPressable` (our forked implementation) or `usePressableState` (reuses the implementation in React Native) ?  The answer informs future PR comments :)

Reply:
I believe I'm intending to use our forked implementation of useAsPressable. I'll change Link to use this as well.